user-journey.adoc

```diff
@@ -75,25 +74,28 @@ sudo systemctl daemon-reload
 sudo systemctl restart docker
 ----
 .. Mac OSX
-... Configure a CentOS VM using as Hypervisor virtualbox or xhyve. The system will require at least 6 Gb of RAM and 4 CPUs - Process has been documented here: https://gist.github.com/cmoulliard/039e85e17ee99ad8c9e25b975d3f3a8a 
+... Configure a CentOS VM using as Hypervisor virtualbox or xhyve. The system will require at least 6 Gb of RAM and 4 CPUs - Process has been documented here: https://gist.github.com/cmoulliard/039e85e17ee99ad8c9e25b975d3f3a8a
 ... Next, the docker daemon and Red Hat certificates should be installed using the same instructions for CentOS/Fedora using yum or dnf (see 1. B., 3. a.).
```

Comment:
(see 1. B., 3. a.).   Not clear if you're referring to this set of instructions?  If so, then the numbering probably needs to be updated to match the new numbering in this PR.

Reply:
Just removing it. I think it's clear enough without it.